tsl/src/data_node.c

```diff
@@ -509,8 +510,10 @@ data_node_bootstrap_extension(TSConnection *conn)
 		}
 
 		remote_connection_cmdf_ok(conn,
-								  "CREATE EXTENSION " EXTENSION_NAME " WITH SCHEMA %s CASCADE",
-								  schema_name_quoted);
+								  "CREATE EXTENSION " EXTENSION_NAME
+								  " WITH SCHEMA %s VERSION '%s' CASCADE",
+								  schema_name_quoted,
+								  ts_extension_get_version());
```

Comment:
We use this everywhere else, even for constants.
```suggestion
								  quote_literal_cstr(ts_extension_get_version()));
```

Reply:
I already quoted in the format string, but can change to this approach if preferred.

Reply:
Yes, I noted that you already had quotes and I don't expect that there is a problem with this code, but I think we should always use the quoting functions where appropriate. It reduces the mental burden on both developers, reviewers, and maintainers to not have exceptions from this and is a simple way to reduce the risk of an SQL injection even when working with the code. We can also automate checks that quoting is used correctly, if necessary.